### DISCUSSION TITLE
Example or documentation for setting up RTMP Authentication for OME

### DISCUSSION THREAD

**Discussion by scotthooten** [OvenMediaLabs/OvenMediaEngine#1628] [Help]
Hello,

I posted this question before but for some reason the post appears to be missing so I'm writing another one.

Does anyone have an example of setting up RTMP Authentication for OME?  Or documentation that shows how to set it up correctly as it differs from wowzas method.  We are currently trying to replace wowza with OME, and require RTMP authentication and we don't know how to set it up.   I have also tried emailing Airensoft, and DMing a few people in the forums, but hadnt heard back from anyone.  So any help that can be provided is greatly appreciated. 

Thank you in advance.

**Comment by naanlizard** [comment]
I'm on my phone so it's not an in depth answer, but basically you need to use ingest callbacks to have ome query your authentication method

**Reply by scotthooten** [reply]
Thank you for the reply.  I would love to see if there was some documentation or examples that show that.  Do you know where i can find that?

**Reply by getroot** [reply]
OME does not support ID/Password type RTMP Auth. Instead, like naanlizrd said, try using AdmissionWebhooks.
https://airensoft.gitbook.io/ovenmediaengine/access-control/admission-webhooks
